x-pack/plugins/monitoring/public/services/enable_alerts_modal.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ajaxErrorHandlersProvider } from '../lib/ajax_error_handler';
+import { showAlertsToast } from '../alerts/lib/alerts_toast';
+
+export function enableAlertsModalProvider($http, $window, $injector) {
+  function shouldShowAlertsModal(alerts) {
+    const modalHasBeenShown = $window.sessionStorage.getItem('ALERTS_MODAL_HAS_BEEN_SHOWN');
```

Comment:
@estermv Is it on purpose that you use `sessionStorage` here instead of `localStorage` as you do later on for decisionMade?

Reply:
Yes, it's to easily manage the `Remind me later` option. The modal will appear again to the user in the next page session. This gives a nice "remind me later" feeling on when to show the modal again. (We can always add a specific timer later if we see this don't work)

When the user selects `Yes` or `No` we don't want the modal to appear again, so I use the localStorage as "permanent" storage. (Of course, there are some edge cases, like the user cleaning the localStorage or using a different browser, but we left this out from the scope of this PR)

Reply:
I believe the `sessionStorage` is for when you click "Remind me later".